libs/liballoc/liballoc.c

```diff
@@ -2,112 +2,98 @@
 
 /**  Durand's Amazing Super Duper Memory functions.  */
 
-#define VERSION     "1.1"
-#define ALIGNMENT   16ul//4ul               ///< This is the byte alignment that memory must be allocated on. IMPORTANT for GTK and other stuff.
-
-#define ALIGN_TYPE      char ///unsigned char[16] /// unsigned short
-#define ALIGN_INFO      sizeof(ALIGN_TYPE)*16   ///< Alignment information is stored right before the pointer. This is the number of bytes of information stored there.
+#define VERSION "1.1"
+#define ALIGNMENT                                                                                  \
+    16ul // 4ul               ///< This is the byte alignment that memory must be allocated on.
```

Comment:
Damn, that looks pretty bad :(

Reply:
yeah, I am going to manually fix that, although the content of `libs` is not that important to me.